modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxDDMFormFieldTypeReportProcessorTest.java

```diff
@@ -146,8 +180,30 @@ private void _setUpJSONFactoryUtil() {
 		jsonFactoryUtil.setJSONFactory(new JSONFactoryImpl());
 	}
 
+	private void _setUpLanguageUtil() {
+		when(
+			_language.get(
+				Mockito.any(Locale.class), Mockito.eq(StringPool.FALSE))
+		).thenReturn(
+			"False"
+		);
+		when(
+			_language.get(
+				Mockito.any(Locale.class), Mockito.eq(StringPool.TRUE))
+		).thenReturn(
+			"True"
+		);
+
+		LanguageUtil languageUtil = new LanguageUtil();
+
+		languageUtil.setLanguage(_language);
```

Comment:
@cleydyr I would use the `burrito` rule here.
```
LanguageUtil languageUtil = new LanguageUtil();

when(...)

when(...)

languageUtil.setLanguage(_language);
```

What do you think?

Reply:
Hm... I was thinking about declaring `languageUtil` it as close a possible to its usage instead.